01-react-week-0.5/styling.md

```diff
@@ -114,6 +114,71 @@ What else can we do? How far can we extend this? Everything else about JavaScrip
 - You can add as many classes as you want. If you want a div to have the two classes "content" and "profile", you can say `<div className='content profile'></div>`
 - You can make the value of `className` to be a variable. If you have a variable `let myClass = 'my-cool-class'`, you can say `<div className={myClass}></div>`
 - If you need to conditionally set some styles, you can do that through conditionally modifying a string! You'll have to set a variable to the correct value before returning JSX, because you can't do conditional logic in JSX
+check
+<!--BEGIN CHALLENGE-->
```

Comment:
I think you need a space between `--` and `B` otherwise Learn doesn't know it's a comment. 

```suggestion
<!-- BEGIN CHALLENGE-->
```

Reply:
> I think you need a space between `--` and `B` otherwise Learn doesn't know it's a comment.

huh interesting... I've been using the galvanize snippets and it seems to display correctly in the `learn preview`, but I'll keep an eye on that